webpack/components/TimelineScrubber.jsx

```diff
@@ -93,6 +94,35 @@ class TimelineScrubber extends Component {
     this.resetScrubber();
   }
 
+  handleMouseWheel(event) {
+    const wheelDown = debounce(() => {
+      this.setState({
+        progress: clamp(this.state.progress - 0.02, 0, 1)
+      });
+      this.updateTime();
+    }, 5);
+
+    const wheelUp = debounce(() => {
+      this.setState({
+        progress: clamp(this.state.progress + 0.02, 0, 1)
+      });
+      this.updateTime();
+    }, 5);
+
+    event.persist();
+    const { deltaX, deltaY } = event.nativeEvent;
+    const [dominant] = [deltaX, deltaY].sort(
```

Comment:
Clever, but I agree that behavior-wise, it feels more natural to have `[deltaX, -deltaY]` here.

Reply:
Mouse/trackpad video scrubber deep dive (all on my MacBook Pro):

**Mouse**
_Roll up_ = rotate top of mouse wheel away from me (causes most doc viewers to “**scroll down**” towards the bottom of the page)
_Roll down_ = rotate top of mouse wheel towards me/the bottom of the mouse (causes most doc viewers to “**scroll up**” towards the top of the page)

**Trackpad**
_Slide up_ = slide two fingers towards the screen (note that this causes most doc viewers to “**scroll down**” towards the bottom of the page)
_Slide down_ = slide two fingers (causes doc viewers to “**scroll up**” towards the top of the page)

Confirmed: _rolling up_ the mouse wheel and _sliding up_ the trackpad produces the same result, typically **scrolling down** the page, whereas _rolling down_ the mouse wheel and _sliding down_ the trackpad both cause the page to **scroll up**.

Assumption: the lateral sliding behavior with **deltaX** on trackpad is preferred: sliding left moves the scrubber left (backs up the video), sliding right moves the slider right (advances the video)

**[deltaX, deltaY]**
Mouse with wheel:
Roll up = video backs up
Roll down = video advances
Trackpad:
Slide up = video backs up
Slide down = video advances

**[deltaX, -deltaY]**
Mouse with wheel:
Roll up: video advances
Roll down: video backs up
Trackpad:
Slide up: video advances
Slide down: video backs up
** This behavior is preferable to me, but as noted above, it contradicts the now-customary behavior of document viewers, prompted by the rise of touchscreens, whereby rolling/sliding up moves the page down (advances) and vice versa. Therefore [deltaX, deltaY] may be more intuitive to most users, assuming the metaphor of “scrolling down a page = advancing forward in time”. My preference for the inverted option is a bit hard to explain, but possibly the result of a youth misspent playing flight simulators.